microservices/product-search-export/docker/Dockerfile

```diff
@@ -156,48 +139,106 @@ RUN GPG_KEYS=B0F4253373F8F6F510D42178520A9993A1C052F8 \
     && ln -sf /dev/stderr /var/log/nginx/error.log
 
 COPY docker/nginx/nginx.conf /etc/nginx/nginx.conf
-COPY docker/nginx/nginx-site.conf /etc/nginx/conf.d/nginx-site.conf
 
 EXPOSE 80
 
 STOPSIGNAL SIGTERM
 
+ENV ELASTICSEARCH_HOSTS_STRING='elasticsearch:9200'
+ENV APP_SECRET=bef043dff42c07c95949d8bfddf1081f
+
+# install necessary PHP extension
+RUN docker-php-ext-install \
+    opcache
+
+# install locales and switch to en_US.utf8 in order to enable UTF-8 support
+# see https://github.com/docker-library/php/issues/240#issuecomment-305038173
+RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+    gnu-libiconv \
+    sudo
+
+ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
+ENV LC_ALL=en_US.utf8 LANG=en_US.utf8 LANGUAGE=en_US.utf8
+
+COPY docker/nginx/nginx-site.conf /etc/nginx/conf.d/nginx-site.conf
+COPY docker/php-ini-overrides.ini /usr/local/etc/php/php.ini
+
+# overwrite the original entry-point from the PHP Docker image with our own
+COPY docker/docker-php-entrypoint /usr/local/bin/
+
 # nginx and php-fpm need to be started under root privileges after the container gets started
 # since our image runs under the user www-data, we need to modify sudoers (exceptions to sudo permission)
 # this allows the user www-data to execute both commands with sudo without the need of providing password
-RUN echo "www-data ALL=(ALL) NOPASSWD: /usr/sbin/nginx, /usr/local/sbin/php-fpm" >> /etc/sudoers
+RUN echo "www-data ALL=(ALL) NOPASSWD:SETENV: /usr/sbin/nginx, /usr/local/sbin/php-fpm" >> /etc/sudoers
```

Comment:
maybe the use of SETENV could be described since there was quite problem with that